es.go

```diff
@@ -80,6 +80,32 @@ type ShardOverlap struct {
 	ReplicasTotal int
 }
 
+//Holds information about shard recovery based on the _cat/recovery API: https://www.elastic.co/guide/en/elasticsearch/reference/5.6/cat-recovery.html
+type ShardRecovery struct {
+	Index                string `json:"index"`
+	Shard                string `json:"shard"`
+	Time                 string `json:"time"`
```

Comment:
It would be nice to get this as a `time.Duration` if possible.

Reply:
👍 Great idea.  I've implemented this as `ShardRecovery.TimeRemaining()` in my latest push.